src/api/generate-xlsx/styled-columns/is-title-row/index.test.ts

```diff
@@ -0,0 +1,56 @@
+import isTitleRow from '.';
+import SECTION_NAMES from '../../../constants/XLSX-CONFIG/SECTION_NAMES';
+import APPLICATION_INFORMATION_INDEXES from '../../../constants/XLSX-CONFIG/INDEXES/APPLICATION_INFORMATION';
+
+const { APPLICATION_INFORMATION } = SECTION_NAMES;
+const { EXPORTER_CONTACT_DETAILS, KEY_INFORMATION } = APPLICATION_INFORMATION_INDEXES;
+
+describe('api/generate-xlsx/styled-columns/is-title-row', () => {
+  describe(`when the sheetName is ${APPLICATION_INFORMATION}`, () => {
+    const mockSheetName = APPLICATION_INFORMATION;
+
+    describe(`when rowNumber=${EXPORTER_CONTACT_DETAILS}`, () => {
+      it('should return true', () => {
+        const result = isTitleRow(mockSheetName, EXPORTER_CONTACT_DETAILS);
+
+        expect(result).toEqual(true);
+      });
+    });
+
+    describe(`when rowNumber=${KEY_INFORMATION}`, () => {
+      it('should return true', () => {
+        const result = isTitleRow(mockSheetName, KEY_INFORMATION);
+
+        expect(result).toEqual(true);
+      });
+    });
+
+    describe('when rowNumber is NOT 8 or 13', () => {
+      it('should return false', () => {
+        const result = isTitleRow(mockSheetName, 5);
+
+        expect(result).toEqual(false);
+      });
+    });
+  });
+
+  describe(`when the sheetName is NOT ${APPLICATION_INFORMATION}, rowNumber=1`, () => {
```

Comment:
, rowNumber=1 does not make sense in this describe since we have `when rowNumber is NOT 1`?

Reply:
Whoops, have removed from the describe.